examples/detection/eval_coco_multi.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+
+import chainer
+from chainer import iterators
+
+import chainermn
+
+from chainercv.datasets import coco_bbox_label_names
+from chainercv.datasets import COCOBboxDataset
+from chainercv.evaluations import eval_detection_coco
+from chainercv.links import FasterRCNNFPNResNet101
+from chainercv.links import FasterRCNNFPNResNet50
+from chainercv.utils import apply_to_iterator
+from chainercv.utils import ProgressHook
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--model',
+        choices=('faster_rcnn_fpn_resnet50', 'faster_rcnn_fpn_resnet101'),
+        default='faster_rcnn_fpn_resnet50')
+    parser.add_argument('--pretrained-model')
+    parser.add_argument('--batchsize', type=int, default=1)
+    args = parser.parse_args()
+
+    if args.model == 'faster_rcnn_fpn_resnet50':
```

Comment:
I prefer reusing part of `eval_coco.py`.

Reply:
I'll do this in another PR. It will also merge `eval_voc07`and `eval_coco` like `eval_semantic_segmentation`.